File: Sean的leetcode/python/Excel Sheet Column Title.py
class Solution(object):
    def convertToTitle(self, n):
        """
        :type n: int
        :rtype: str
        """
        re = ""
        while n:
            n -= 1
            re = chr(ord('A') + n % 26) + re
            n /= 26
        return re

File: MyLeetCode/Leetcode/Excel Sheet Column Title.cpp
#if 1 //recursive solution
class Solution {
public:
    string convertToTitle(int n) {
        return (n == 0) ? ("") : (convertToTitle((n-1) / 26) + (char)((n - 1) % 26 + 'A'));
    }
};
#else
class Solution {
public:
    string convertToTitle(int n) {
        string res;
        while (n)
        {
            res = (char)(((n-1) % 26) + 'A') + res;
            n = (n - 1) / 26;
        }
        return res;
    }
};
#endif